extensions/amp-story/1.0/pagination-buttons.css

```diff
@@ -59,9 +59,14 @@ amp-story:not([desktop]) .i-amphtml-story-button-container {
   pointer-events: none !important;
 }
 
-/* Navigation buttons. These are only displayed on desktop. */
-.i-amphtml-story-button-move {
-  display: none !important;
+/* Navigation button element. Visibly hidden on mobile but focusable by screen readers. */
```

Comment:
Are they click targets that could cover some content? Should we add `pointer-events: none`? That might create some P0s.

Reply:
The click targets could cover content but they're within the 48px area reserved for navigating.
I don't think we need `pointer-events: none` since all clicks within the 48px area are ignored.
<img width="455" alt="Screen Shot 2021-02-25 at 11 51 19 AM" src="https://user-images.githubusercontent.com/3860311/109187316-c83d6f80-775f-11eb-8913-aeecca08f36e.png">